core/bytes/bytes.odin

```diff
@@ -334,52 +334,126 @@ Inputs:
 Returns:
 - index: The index of the byte `c`, or -1 if it was not found.
 */
-index_byte :: proc(s: []byte, c: byte) -> (index: int) #no_bounds_check {
-	length := len(s)
-	i := 0
-
-	// Guard against small strings.
-	if length < SIMD_SCAN_WIDTH {
-		for /**/; i < length; i += 1 {
+index_byte :: proc "contextless" (s: []byte, c: byte) -> (index: int) #no_bounds_check {
```

Comment:
These procs have been made contextless but not the ones in `core:strings`. Why is that?

Reply:
More things in core can be made contextless, but I reverted this for now to delay it for a later day when a pass can be done over all the things.